app/src/androidTest/java/SettingsActivityTest.java

```diff
@@ -17,13 +17,13 @@
     public ActivityTestRule<SettingsActivity> settingsActivityActivityTestRule =
             new ActivityTestRule<>(SettingsActivity.class);
 
-    @Test
```

Comment:
Test class SettingsActivity has been removed from the project. Moving forward, tests classes will mention the feature they test. This might be the source of conflict.

Reply:
Gradle file onflicts resolved.